services/close_order_list.rb

```diff
@@ -0,0 +1,13 @@
+class CloseOrderList
+
+  def self.call
+    order_list = OrderList.last
+    orders = order_list.show_orders
+    if order_list.closed
+      "Orders already closed!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.